### ISSUE TITLE
Remaining NEON edits / changes

### ISSUE THREAD

**Issue by aebudden** [NCEAS/nceas-training#73]
- General: All external links to open in a new window
- Section 1.1: Video frames not rendering in online. Fine in local copy. Video in Section 4.4 is fine both online and in local build
- Section 3.1.3.2: Have paragraphs start under the bold headers
- Section 3.1.4: Create visual separation between the two exercises
- Section 3.1.4: Horizontal division bar is sitting within the exercise box
- Section 3.3.2.7: Add some buffer between the RHS of the image and the text
- Section 3.3.2.9: Missing image / code error
- Section 3.3.2.9: Image credit separated from image location
- Section 3.4.1: Capitalize first title word
- Section 3.4.3: Typo 'resources' under DLC image

**Comment by ggwillard** [comment]
Section 3.1.4.2 (bottom of the green "exercise" box): typo, should say "Thinking" not Tinking

**Comment by brunj7** [comment]
- [ ] Section `about this resource`: R version is outdated,  bump it to the latest 4.0.3
- [ ] Section 1.1: the videos are not showing up (it does on the local version, might be a github pages thing)
- [ ] Section 2.1.2:  ...activity `Inclusion` <- fix case
- [ ] Section 2.1.2, under `Explore alignment`: I think there is a word missing in this sentence: `There are many online resources available to help, however you may also with to talk to individuals within those positions.`
- [ ] Section 3.1.2: add reference to section 3.4 at the end

**Comment by aebudden** [comment]
@brunj7 Are you able to problem solve 1.1?

**Comment by brunj7** [comment]
@aebudden yes! 1.1 and  I can go through the edits in 1-hour (have some calls ahead). Please keep the checkboxes updated if you have time to tackle some before

**Comment by brunj7** [comment]
most of those edits have been implemented by #78 

Note that I am not aware of a way to make all the links open into a new window/tab
